tests/helpers.test.ts

```diff
@@ -1,71 +1,91 @@
-    calculateSER
-} from '../src/helpers';
+    TranscriptionAnalyzer
+} from '../src/TranscriptionAnalyzer';
 
-import { TranscriptionAnalysisService } from '../src/TranscriptionAnalysisService';
+import { TranscriptionFileService } from '../src/TranscriptionFileService';
 import { TestResult } from '../src/types';
 
-const analyzer = new TranscriptionAnalysisService();
-
 const fakeFileContent = 'sample.wav\tthis is only a sample';
 const fakeAudioFolder = './audio-recording-info'
-const fakeItemInResult = { recording: path.join(fakeAudioFolder, fakeFileContent.split('\t')[0]), transcription: fakeFileContent.split('\t')[1]}
-const testData = parseTextFileContent(fakeFileContent, fakeAudioFolder)
-const TestResultsMock = jest.fn<TestResult, any[]>(()=> ({
-    actualTranscription:'this is only a sample',
+
+const fakeItemInResult = {
+    recording: path.join(fakeAudioFolder, fakeFileContent.split('\t')[0]),
+    transcription: fakeFileContent.split('\t')[1]
+};
+
+const tfs = new TranscriptionFileService();
+const testData = tfs.parseTextFileContent(fakeFileContent, fakeAudioFolder);
+
+const TestResultsMock = jest.fn<TestResult, any[]>(() => ({
+    actualTranscription: 'this is only a sample',
     expectedTranscription: 'this is only a sample',
     wordErrorRate: .023
 }));
-const calculateSERMock = jest.fn<string, any[]>(calculateSER);
+
+const transcriptAnalyzer = new TranscriptionAnalyzer();
+
+const responseAnalyzer = new ResponseAnalyzer(transcriptAnalyzer);
+
+const calculateSERMock = jest.fn<string, any[]>(responseAnalyzer.calculateSER);
+
 
 describe('validateExpectedTranscription', () => {
     it('Throws an error when transcription is invalid', () => {
-        expect( ()=>{
-            analyzer.validateExpectedTranscription('I\'am not, ready')
+        expect(() => {
+            transcriptAnalyzer.validateExpectedTranscription('I\'am not, ready')
         }).toThrowError(SyntaxError);
     });
 
     it('Not Throw an error when transcription is valid', () => {
         expect(() => {
-            analyzer.validateExpectedTranscription('I\'am not ready')
+            transcriptAnalyzer.validateExpectedTranscription('I\'am not ready')
         }).not.toThrowError(SyntaxError);
     });
 })
 
 describe('cleanExpectedTranscription', () => {
     it('Replaces hyphens with space', () => {
-        expect(analyzer.cleanExpectedTranscription('test-harness')).toEqual('test harness');
+        expect(transcriptAnalyzer.cleanExpectedTranscription('test-harness'))
+            .toEqual('test harness');
     });
 
     it('Lowercases strings', () => {
-        expect(analyzer.cleanExpectedTranscription("THIS IS SPARTA")).toStrictEqual('this is sparta');
+        expect(transcriptAnalyzer.cleanExpectedTranscription("THIS IS SPARTA"))
+            .toStrictEqual('this is sparta');
     });
 })
 
 
-describe('createTestData', ()=>{
+describe('createTestData', () => {
     it('Throws an error when file path is undefined', () => {
```

Comment:
Why? do we not support relative paths?

Reply:
all file paths are supported... undefined should NOT be.